contracts/core/tokens/rebalancing-libraries/StandardProposeLibrary.sol

```diff
@@ -84,6 +84,12 @@ library StandardProposeLibrary {
             "RebalancingSetToken.propose: State must not be Rebalance"
         );
 
+        // New proposal cannot be made during drawdown
```

Comment:
Hmm I disagree here because there are multiple states it CAN work in. So I feel like it'd be more informative to tell them the specific state it's in that doesn't make it work. Especially since Drawdown essentially means the death of the token, I'd prefer to inform users that's the case (which might not be relevant for `propose` but is for `mint`.